catalog/app/utils/BucketPreferences/BucketPreferences.ts

```diff
@@ -39,6 +39,7 @@ interface BlocksPreferencesInput {
   code?: boolean
   meta?: boolean | MetaBlockPreferencesInput
   gallery?: boolean | GalleryPreferences
+  qurator?: boolean
```

Comment:
do you think we need this both in global and per-bucket configs?

Reply:
We have to have it in global config.
It is in a bucket config for the consistency: all other sections on the page can be turned off using per-bucket config.